octodns_powerdns/__init__.py

```diff
@@ -683,6 +780,37 @@ def _apply(self, plan):
         try:
             self._patch(f'zones/{encoded_name}', data={'rrsets': mods})
             self.log.debug('_apply:   patched')
+
+            # update meta
+            _data = {}
+
+            desired_master_tsig_key_ids = plan.meta.get(
+                'master_tsig_key_ids', {}
+            ).get('desired', None)
+
+            if desired_master_tsig_key_ids is not None:
+                _data["master_tsig_key_ids"] = desired_master_tsig_key_ids
+                self.log.info(
+                    '_apply: make change to master_tsig_key_ids meta on %s',
```

Comment:
Minor grammatical nit. 

```suggestion
                    '_apply: making a change to master_tsig_key_ids meta on %s',
```